api/consumers.py

```diff
@@ -253,24 +262,45 @@ async def receive(self, text_data):
     async def submit_response(self, data):
         response = await self.create_user_response(data)
 
-        await self.channel_layer.group_send(
-            f"quiz_session_instructor_{self.code}",
-            {
-                "type": "update_answers",
-                "question_id": response["question_id"],
-            },
-        )
-        await self.channel_layer.group_send(
-            f"quiz_session_instructor_{self.code}",
-            {
-                "type": "user_response",
-                "response": data.get("data").get("selected_answer"),
-            },
-        )
+        if "type" not in response:
```

Comment:
I'm not a fan of this; how about we do `if response.get("status") == "success":` and update `create_user_response` so that it includes a `status`?

Reply:
Done, this is definitely a more maintainable approach.